iocore/net/quic/QUICStreamState.cc

```diff
@@ -250,31 +250,26 @@ bool
 QUICSendStreamState::is_allowed_to_send(QUICFrameType type) const
 {
   switch (this->get()) {
-  case State::_Init:
-    break;
   case State::Ready:
-    if (type == QUICFrameType::STREAM || type == QUICFrameType::STREAM_BLOCKED || type == QUICFrameType::RST_STREAM) {
-      return true;
-    }
-    break;
   case State::Send:
     if (type == QUICFrameType::STREAM || type == QUICFrameType::STREAM_BLOCKED || type == QUICFrameType::RST_STREAM) {
       return true;
     }
     break;
+  /*A sender MUST NOT send STREAM or STREAM_DATA_BLOCKED after sending a RESET_STREAM; that is, in the terminal states and in the
+   * “Reset Sent” state.*/
+  case State::ResetSent:
```

Comment:
Does this mean we can RST_STREAM multiple times? I'm not sure whether we can do that.